src/Game/UI/Gumps/OptionsGump.cs

```diff
@@ -1921,6 +1921,25 @@ private void Apply()
             Engine.Profile.Current.ShowTargetRangeIndicator = _showTargetRangeIndicator.IsChecked;
 
             Engine.Profile.Current.CustomBarsToggled = _CustomBars.IsChecked;
+            var hbgstandard = Engine.UI.Gumps.OfType<HealthBarGump>().ToList();
+            var hbgcustom = Engine.UI.Gumps.OfType<HealthBarGumpCustom>().ToList();
+            if (_CustomBars.IsChecked)
+            {               
+                foreach (var healthbar in hbgstandard)
+                {
+                    Engine.UI.Add(new HealthBarGumpCustom(healthbar.LocalSerial) { X = healthbar.X, Y = healthbar.Y });
+                    healthbar.Dispose();
+                }
+            }
+            else if (!_CustomBars.IsChecked)
```

Comment:
a simple else is enough

Reply:
and `_CustomBars` -> `_customBars`

Reply:
Feel free to make changes such as that to fit with your coding style. I like the way it is, functions properly, without issue and is written in a format to where others with less knowledge in coding can understand.